src/neuromaps_prime/transforms/surface.py

```diff
@@ -53,3 +62,438 @@ def surface_sphere_project_unproject(
         raise FileNotFoundError(f"Sphere out not found: {sphere_out}")
 
     return result
+
+
+def metric_resample(
+    metric_in: Path,
+    current_sphere: Path,
+    new_sphere: Path,
+    metric_out: str,
+    method: str = "ADAP_BARY_AREA",
+    area_surfs: tuple[Path, Path] | None = None,
+) -> workbench.MetricResampleOutputs:
+    """Resample a surface metric from one sphere to another.
+
+    Parameters
+    ----------
+    metric_in : Path
+        Path to input metric file.
+    current_sphere : Path
+        Path to current spherical surface.
+    new_sphere : Path
+        Path to new spherical surface.
+    metric_out : str
+        Path to output metric file.
+    method : str, optional
```

Comment:
Nice, for the final version we should enforce sampling methods. Also double check, but with niwrap, these resampling methods might be a workbench.metric_resample_param.